### DISCUSSION TITLE
DLNA access

### DISCUSSION THREAD

**Discussion by sunjaxx** [ampache/ampache#2930] [Q&A]
Hello!

I am (very) new to unix and docker. I just installed ampache (docker), and it's working fine from the web interface. I would now like to enable the dlna service. I found these instructions:

1. You must enable UPnP Backend in System settings.
2. You must also enable Enable url rewriting in Streaming settings.
3. If your installation is within a subfolder '/ampache' you must take care for your Rewrite Rules within '[ampachepath]/play/.htaccess'.
4. PHP Socket mod must be enabled and PHP XMLReader extension available
5. You should broadcast the server periodically calling http://localhost/ampache/upnp/?btnSend=Send+SSDP+broadcast or php bin/broadcast.inc with a cron (every 1-2 minutes recommended).
6. You need to set the http_host option within config/ampache.cfg.php to broadcast via broadcast.inc.
7. You need to enable (remove leading semicolon) ;websocket_address = "ws://localhost:8100" in config/ampache.cfg.php.

I did 1, 2 and 7, but I don't really understand 3, 4 ,5 and 6. Are those relevant for a docker installation as well? Could someone please elaborate a bit more on those? Thanks!

**Comment by cnic023** [comment]
The instructions are very vague and ambiguous which is why I can't get all my ducks in a row! 5 is necessary so the server can broadcast its presence on the network to establish a UPnP connection. Oddly this is done by a crontab entry. You have a choice of what should go there. It may be a php command to ampache/bin/broadcast.inc or it can be a direct call to broadcast to localhost. 
The first option can be tried from the command line with: "/usr/bin/php -f [path to ampache]/bin/broadcast.inc" You should get a response to show the broadcast has been made.
Sorry but I'm as much in the dark as you! See my question in 'discussions'

**Reply by cvinhaes** [reply]
+1 with the same issue here.
